src/main/java/nl/miraclethings/tools/data/DateUtil.java

```diff
@@ -19,9 +19,15 @@ public static Date getBaseDate(Date date) {
         return getBaseDate(date, Calendar.DATE);
     }
 
-    public static Date getBaseDate(Date date, int dateType) {
+    Date getBaseDate(Date date, int dateType) {
```

Comment:
Misschien is het wel handig om je codewijzigingen ook te testen voordat je een PR maakt :-/